backend/src/main/java/de/neuefische/backend/model/openai/ChatGptResponse.java

```diff
@@ -0,0 +1,34 @@
+package de.neuefische.backend.model.openai;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatGptResponse {
+    /**
+     {
+
+     "choices":
+     [
+     {
+     "message": {
+     "role": "assistant",
+     "content": "This is a test!"
+     }
+     }
+     ]
+     }
+     */
```

Comment:
you dont want comments in your production code

Reply:
done!